src/networks.json

```diff
@@ -398,6 +398,20 @@
     "explorer": "https://scan.meter.io",
     "start": 4992871,
     "imageIPFS": "QmSZvT9w9eUDvV1YVaq3BKKEbubtNVqu1Rin44FuN4wz11"
+  },
+    "87": {
+    "key": "87",
+    "name": "Nova Network",
+    "shortName": "Nova",
+    "chainId": 87,
+    "network": "mainnet",
+    "multicall": "0x12C8e061782A011cB3Fe54fB22DaFf73186EeA6d",
+    "rpc": [
+      "https://connect.novanetwork.io"
```

Comment:
Hi, @nova-network-inc This node is not an archive node, needs a full archive node here, so it can fetch data from older blocks
```
(body="{\"jsonrpc\":\"2.0\",\"id\":44,\"error\":{\"code\":-32000,\"message\":\"missing trie node 5ae03d545de054bb1c646e37a4d9f20ff0d0791e51175c424055f3dfb57fe6c4 (path )\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_getBalance\",\"params\":[\"0x12c8e061782a011cb3fe54fb22daff73186eea6d\",\"0x1\"],\"id\":44,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://connect.novanetwork.io", code=SERVER_ERROR, version=web/5.4.0)

```

Reply:
> Hi, @nova-network-inc This node is not an archive node, needs a full archive node here, so it can fetch data from older blocks
> 
> ```
> (body="{\"jsonrpc\":\"2.0\",\"id\":44,\"error\":{\"code\":-32000,\"message\":\"missing trie node 5ae03d545de054bb1c646e37a4d9f20ff0d0791e51175c424055f3dfb57fe6c4 (path )\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_getBalance\",\"params\":[\"0x12c8e061782a011cb3fe54fb22daff73186eea6d\",\"0x1\"],\"id\":44,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://connect.novanetwork.io", code=SERVER_ERROR, version=web/5.4.0)
> ```

Hi @ChaituVR , I've just changed it to https://rpc.novanetwork.io:9070/ that should do.

Reply:
Hi @nova-network-inc It is tills the same ,
```
 Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":44,\"error\":{\"code\":-32000,\"message\":\"missing trie node 5ae03d545de054bb1c646e37a4d9f20ff0d0791e51175c424055f3dfb57fe6c4 (path )\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_getBalance\",\"params\":[\"0x12c8e061782a011cb3fe54fb22daff73186eea6d\",\"0x1\"],\"id\":44,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://rpc.novanetwork.io:9070/", code=SERVER_ERROR, version=web/5.4.0)
 ```